src/main/java/org/folio/circulation/domain/LoanCheckInService.java

```diff
@@ -16,4 +16,13 @@
       : loan.checkIn(request.getCheckInDate(), systemDateTime,
         request.getServicePointId()));
   }
+
+  public boolean isInHouseUse(Item item, RequestQueue requestQueue,
+    CheckInByBarcodeRequest checkInByBarcodeRequest) {
+
+    return item.isAvailable()
+      && requestQueue.size() == 0
+      && item.getLocation().homeLocationIsServedBy(checkInByBarcodeRequest
+      .getServicePointId());
```

Comment:
Sonar is still reporting partial coverage on this statement. Please could you investigate why?

Reply:
It is covered now https://sonarcloud.io/component_measures?id=org.folio%3Amod-circulation&metric=new_coverage&pullRequest=456&selected=org.folio%3Amod-circulation%3Asrc%2Fmain%2Fjava%2Forg%2Ffolio%2Fcirculation%2Fdomain%2FLoanCheckInService.java&view=list